File: public/javascripts/mobile/app/model/News.js
Ext.define('Genesis.model.News',
{
   extend : 'Ext.data.Model',
   id : 'News',
   alternateClassName : 'News',
   config :
   {
      idProperty : 'id',
      fields : ['id', 'item_id', 'title', 'text', 'type', 'item_type', 'photo',
      {
         name : 'created_date',
         type : 'date',
         convert : function(value, format)
         {
            value = Date.parse(value, "yyyy-MM-dd");
            return (value) ? Genesis.fn.convertDateNoTimeNoWeek(value) : null;
         }
      }]
   }
});

File: app/models/news.rb
class News
  attr_accessor :type, :item_id, :item_type, :title, :photo, :text
  
  def initialize(hash)
    hash.keys.each do |key|
      m = "#{key}="
      self.send( m, hash[key] ) if self.respond_to?( m )
    end
  end
end